### ISSUE TITLE
cuatrimoto.html

### ISSUE THREAD

**Issue by LeidyKCastaneda** [LeidyKCastaneda/Reto2#5]
<!DOCTYPE html>
<html>
<head>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>    
    <script type="text/javascript" src="./funcionescuatrimoto.js"></script>
    <title>Reto2</title>
</head>
<body> 
    <h1>Registro de cuatrimotos</h1>
    <p id="texto"></p>
    <div id="respuestaCuatrimoto"></div>
        <label> ID de cuatrimoto</label>
        <input type="number" placeholder=" " id="ID" /><br>
        <label> Marca </label>
        <input type="text" placeholder=" " id="BRAND" /><br>
        <label> Modelo</label>
        <input type="number" placeholder=" " id="MODEL" /><br>
        <label> Categoria</label>
        <input type="text" placeholder=" " id="CATEGORY_id" /><br>   
        <label> Nombre</label>
        <input type="text" placeholder=" " id="NAME" /><br>     
        <button onclick="consultarQuadbike()">Consultar</button>
        <button onclick="updateQuadbike()">Actualizar</button>
        <button onclick="guardarQuadbike()">Guardar</button>
    </div>

 

    
</body>
</html>
